modules/tensor_mechanics/doc/content/modules/tensor_mechanics/beam_vandv.md

```diff
@@ -0,0 +1,254 @@
+
+!listing euler_finite_rot_y.i
+
+### Results
+
+For this beam, the dimensionless parameter alpha is given by:
+\begin{equation}
+alpha = \frac {kAGL^2}{EI} = 1e6
+\end{equation}
+ Since, the value of alpha is quite high, the beam behaves like a thin beam where shear effects are not significant. The analytical solution for the displacements due to large rotations and large strain of the cantilever beam in X and Y direction is -1 m and 2.4 m respectively ([!citet](bishopanddrucker1945)).
+
+The displacement at the free end of the cantilever beam obtained from the MOOSE  is -0.954 m in X direction as shown in [fig:euler_finite_rot_x] and 2.37 m in Y direction as shown in [fig:euler_finite_rot_y].
+
+
+!media media/tensor_mechanics/vandv/euler_finite_rot_x2.png
+       style=width:450px;margin-left:110px;float:center;
+       id=fig:euler_finite_rot_x
+       caption=Displacement of the cantilever beam with large strain and rotation in X direction
+
+
+!media media/tensor_mechanics/vandv/euler_finite_rot_y2.png
+       style=width:450px;margin-left:110px;float:center;
+       id=fig:euler_finite_rot_y
+       caption=Displacement of the cantilever beam with large strain and rotation in Y direction
+
+
+## Torsion
+
+A 1D cantilever beam of 1 m is modeled using beam elements in MOOSE. The beam has a cross section area of 0.5 ${m}^2$, moment of inertia about Y and Z axis of 1e-5 ${m}^4$, Young's modulus of elasticity of 2e9 N/${m}^2$, poisson's ratio of 0.3 and shear coefficient of 1. A torsion of magnitude 5 N/m is applied at the free end of the cantilever beam.
+
+!listing torsion_1.i
+
+### Results
+
+The analytical solution for the axial twist at the free end of the beam is given by:
+\begin{equation}
+\phi = \frac {TL}{GI_{x}} = 3.25e-4 rad
+\end{equation}
+\begin{equation}
+where, I_{x} = I_{y} + I_{z} = 2e-5 m^4
+\end{equation}
+
+The rotations at the free end of the beam obtained in MOOSE is 3.2e-4 rad as shown in [fig:torsion2]
+
+!media media/tensor_mechanics/vandv/torsion2.png
+       style=width:450px;margin-left:110px;float:center;
+       id=fig:torsion2
+       caption=Rotation of the beam due to the applied torsion.
+
+## Small strain Euler beam vibration
+
+A 1D cantilever beam of 4 m is modeled using beam elements in MOOSE. The beam has a cross section area of 0.01 ${m}^2$, moment of inertia about Y and Z axis of 1e-4 ${m}^4$, Young's modulus of elasticity of 1e4 N/${m}^2$, poisson's ratio of -0.99, density of 1 kg/${m}^3$, shear modulus of 4e7 N/${m}^2$ and shear coefficient of 1. An impulse load with a peak load of 0.01 N at 0.05 s as shown in [fig:impulseload] is applied at the free end of the beam in Y direction. The Newmark time integration parameters used in the problem corresponds to the Newmark's average acceleration method, i.e `beta=0.25` and `gamma=0.5`.
```

Comment:
Subject-verb agreement problem in the last sentence (`parameters` and `corresponds`), and I would prefer to see the values used in the problem (are the coefficient examples given the values used or just examples that would be allowable?) Note this problem shows up in many of the results sections on this page. If you fix this paragraph, please fix the others as well.

Reply:
Those are the constant parameters for the Newmark's average acceleration method.